web/src/narrow_error.ts

```diff
@@ -0,0 +1,26 @@
+import render_empty_feed_notice from "../templates/empty_feed_notice.hbs";
+
+interface QueryWord {
+    query_word: string;
+    query: string;
```

Comment:
This should be `is_stop_word: boolean`. I think there isn't a property called `query` for `QueryWord.